### ISSUE TITLE
WebUI for Rucio PROD broken

### ISSUE THREAD

**Issue by dciangot** [dmwm/CMSRucio#425]
WebUI looks currently unavailable, with error (*). Look releted in time with the recent upgrade to 1.30. Need investigation. @ericvaandering any clue?

```
[Wed Feb 22 10:47:57.887776 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220] 2023-02-22 10:47:57,887\trucio.web.ui.flask.main\t21\tERROR\tException on /did [GET]
[Wed Feb 22 10:47:57.887808 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220] Traceback (most recent call last):
[Wed Feb 22 10:47:57.887812 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
[Wed Feb 22 10:47:57.887820 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     response = self.full_dispatch_request()
[Wed Feb 22 10:47:57.887822 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
[Wed Feb 22 10:47:57.887825 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     rv = self.handle_user_exception(e)
[Wed Feb 22 10:47:57.887828 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
[Wed Feb 22 10:47:57.887830 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     rv = self.dispatch_request()
[Wed Feb 22 10:47:57.887832 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
[Wed Feb 22 10:47:57.887835 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
[Wed Feb 22 10:47:57.887837 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/rucio/web/ui/flask/bp.py", line 142, in <lambda>
[Wed Feb 22 10:47:57.887840 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     return lambda: authenticate(template=template, title=title)
[Wed Feb 22 10:47:57.887842 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/rucio/web/ui/flask/common/utils.py", line 595, in authenticate
[Wed Feb 22 10:47:57.887844 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     valid_token_dict = validate_webui_token()
[Wed Feb 22 10:47:57.887847 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/rucio/web/ui/flask/common/utils.py", line 224, in validate_webui_token
[Wed Feb 22 10:47:57.887849 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     valid_token_dict = auth.validate_auth_token(session_token)
[Wed Feb 22 10:47:57.887851 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/rucio/db/sqla/session.py", line 430, in new_funct
[Wed Feb 22 10:47:57.887854 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     result = function(*args, session=session, **kwargs)
[Wed Feb 22 10:47:57.887856 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/rucio/api/authentication.py", line 298, in validate_auth_token
[Wed Feb 22 10:47:57.887858 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     auth = authentication.validate_auth_token(token, session=session)
[Wed Feb 22 10:47:57.887861 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/rucio/db/sqla/session.py", line 444, in new_funct
[Wed Feb 22 10:47:57.887863 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     result = function(*args, session=session, **kwargs)
[Wed Feb 22 10:47:57.887865 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]   File "/usr/local/lib/python3.6/site-packages/rucio/core/authentication.py", line 462, in validate_auth_token
[Wed Feb 22 10:47:57.887868 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220]     raise CannotAuthenticate("No token was passed!")
[Wed Feb 22 10:47:57.887870 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220] rucio.common.exception.CannotAuthenticate: Cannot authenticate.
[Wed Feb 22 10:47:57.887876 2023] [wsgi:error] [pid 21:tid 140263506642688] [client 10.100.23.0:47220] Details: No token was passed!
```

**Comment by dciangot** [comment]
As per @ericvaandering report:

TL;DR Web UI appears to be back up. Would be helpful if someone expecting data to be there takes a look.

I found a couple of perhaps overlapping problems. First was that the auth server deployment was not finishing although several auth servers were running. I'm not sure why this is.

Second, I tried to roll back the WebUI to 1.29 and that didn't work until I also rolled back the chart to 1.29. I will investigate helm chart changes between 1.29 and 1.30 to see if I can spot the issue. Unless this is easily solved, I would propose to keep running 1.29 WebUI with 1.30 Rucio unless there are problems with that.

**Comment by ericvaandering** [comment]
Fixed
